pkg/util/iptables/iptables.go

```diff
@@ -783,3 +789,76 @@ func isResourceError(err error) bool {
 	}
 	return false
 }
+
+type ParseError interface {
+	Line() int
```

Comment:
Needs a godoc comment, especially whether the first line is 0 or 1

Reply:
Done